holesky/run.sh

```diff
@@ -60,6 +60,21 @@ listQuorums() {
     --quorum-id-list 0
 }
 
+updateSocket() {
+  # we have to pass a dummy quorum-id-list as it is required by the plugin
+  docker run --env-file .env \
+    --rm \
+    --volume "${NODE_ECDSA_KEY_FILE_HOST}":/app/operator_keys/ecdsa_key.json \
+    --volume "${NODE_BLS_KEY_FILE_HOST}":/app/operator_keys/bls_key.json \
+    --volume "${NODE_LOG_PATH_HOST}":/app/logs:rw \
+    ghcr.io/layr-labs/eigenda/opr-nodeplugin:release-0.6.1 \
```

Comment:
Now that we have four commands, I think it's worth factoring out this ghcr address as a variable and use this consistently for each command?

Reply:
fair. let me do it